src/sentry/integrations/message_builder.py

```diff
@@ -87,6 +87,9 @@ def build_attachment_text(group: Group, event: GroupEvent | None = None) -> Any
     if ev_type == "error":
         return ev_metadata.get("value") or ev_metadata.get("function")
     elif ev_type == "transaction":
+        if not event:
+            group = getattr(obj, "group", obj)
```

Comment:
We already have `group` here, may as well just use it directly

Reply:
oop